decidim-pages/spec/features/page_show_spec.rb

```diff
@@ -6,31 +6,62 @@
   include_context "feature"
   let(:manifest_name) { "pages" }
 
-  describe "page show" do
+  let(:title) do
+    {
+      "en" => "Hello world",
+      "ca" => "Hola món",
+      "es" => "Hola mundo"
+    }
+  end
+
+  let(:body) do
+    {
+      "en" => "<p>Content</p>",
+      "ca" => "<p>Contingut</p>",
+      "es" => "<p>Contenido</p>"
+    }
+  end
+
+  describe "page show" do  
     before do
       create(:page, feature: feature, title: title, body: body)
       visit_feature
     end
 
-    let(:title) do
-      {
-        "en" => "Hello world",
-        "ca" => "Hola món",
-        "es" => "Hola mundo"
-      }
+    it "renders the content of the page" do
+      expect(page).to have_title("Hello world")
+      expect(page).to have_content("Content")
     end
+  end
 
-    let(:body) do
-      {
-        "en" => "<p>Content</p>",
-        "ca" => "<p>Contingut</p>",
-        "es" => "<p>Contenido</p>"
-      }
+  describe "page show with comments" do
+    let!(:page_feature) { create(:page, feature: feature, title: title, body: body) }
+    let!(:comments) { 3.times.map { create(:comment, commentable: page_feature) } }
+
+    context "if page is commentable" do
+      before do
+        page_feature.update_attribute(:commentable, true)
```

Comment:
I'd have a `let(:commentable) { true/false }` inside the context and move the before block at the describe.